packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -1135,17 +1137,40 @@ export default class SchedulerTask {
                 payload.metricQuery.additionalMetrics,
                 payload.metricQuery.tableCalculations,
             );
-            await this.googleDriveClient.appendToSheet(
-                refreshToken,
-                spreadsheetId,
-                rows,
-                itemMap,
-                payload.showTableNames,
-                undefined, // tabName
-                payload.columnOrder,
-                payload.customLabels,
-                payload.hiddenFields,
-            );
+            if (payload.pivotConfig) {
```

Comment:
uploadGsheetFromQuery method is used when uploading gsheets from charts , results, or dashboards